src/commands/time/finish.js

```diff
@@ -24,108 +24,121 @@ function controller(t) {
 
         var isBug = e.tp_task && e.tp_task.type === 'bug';
         var isTask = !isBug && e.tp_task;
+        var getResourceType;
```

Comment:
This variable would probably be better named "getResource", seeing is it will actually be a promise which will resolve to the TP entity, not just the _type_ of the TP entity.

Naming is hard.